package.json

```diff
@@ -128,11 +128,19 @@
           "when": "false"
         },
         {
-          "command": "vscode-objectscript.studio.actions",
+          "command": "vscode-objectscript.serverCommands.sourceControl",
```

Comment:
@gjsjohnmurray I don't like "sourceControl" here - maybe call it serverCommands.studioExtension? (The "studio extension" framework is more general than just source control.)

Reply:
@timleavitt with this PR I have implemented separate menus for the Source Control commands and the rest, to replicate how Studio behaves. This command brings up the %SourceMenu menu items, which Studio displays under a menu named "Source Control". The command `vscode-objectscript.serverCommands.other` brings up the items from all other menus for which context=0.

`vscode-objectscript.serverCommands.contextSourceControl` shows the %SourceContext menu's items, and `vscode-objectscript.serverCommands.contextOther` shows all the other ones for which context=1

If you'd like to see this in action you can always get the vsix artifact that our CI pipeline creates whenever I push stuff to this PR:

![image](https://user-images.githubusercontent.com/6726799/86026358-34c46b00-ba27-11ea-93fe-3e9521c7ba48.png)

Install this in your VS Code and give it a try. I'll be glad to more of your feedback.

Reply:
@gjsjohnmurray thanks - that makes more sense now. I'll try it out when I get a chance.